cibyl/orchestrator.py

```diff
@@ -47,8 +47,10 @@ def source_information_from_method(source_method):
     :rtype: str
     """
     source = source_method.__self__
-    info_str = f"source {source.name} of type {source.driver} using method "
-    return info_str+f"{source_method.__name__}"
+    info_str = f"source {source.name} of type {source.driver}"
+    if LOG.getEffectiveLevel() < 20:
```

Comment:
I think it would be clearer to use logging.INFO instead of the integer value

Reply:
Agreed. I'll modify it